spm_results_export.m

```diff
@@ -236,8 +236,23 @@
 
 %-Display webpage
 %==========================================================================
-if(multipleExcursions && exNo >1)
-    [~, webID] = web(fHTML, '-new');
+%Check if we are running from the commandline. If no information about
+%cmdline is stored we are running code without running SPM (i.e. for 
+%tests). We display only if we are not in commandline mode.
+try
+    defaults = spm('Defaults', 'FMRI');

```

Comment:
Is there a specific reason not to use the `spm_default()` function as is usually done in SPM? c.f. for example": https://github.com/spm/spm12/blob/master/spm_spm.m#L512

Reply:
The reason I used this is that if I don't open SPM and just run the tests with SPM in my path the modality might not be set and this can cause errors. This way makes sure the modality is set to 'FMRI'. Beyond that I believe 'spm_defaults()' and 'spm('defaults')' do the same thing.